orchagent/intfsorch.cpp

```diff
@@ -406,6 +423,9 @@ bool IntfsOrch::setIntf(const string& alias, sai_object_id_t vrf_id, const IpPre
             intfs_entry.ref_count = 0;
             intfs_entry.proxy_arp = false;
             intfs_entry.vrf_id = vrf_id;
+            if (!vrf_name.empty()) {
+                intfs_entry.vrf_name = vrf_name;
```

Comment:
You don't have to change the API signature - Use `getVRFname `function. May be refer the comment in function `isInbandIntfInMgmtVrf`

Reply:
sure